README.md

```diff
@@ -99,6 +99,7 @@ use { "johmsalas/text-case.nvim",
     require("telescope").load_extension("textcase")
   end,
   keys = {
+    "ga", -- Default invocation prefix
```

Comment:
Seems like it wasn't only happening in my local setup

Would you remove the note in line 94?
`Author's Note: If default keymappings fail...`

Can't provide the link rn because I'm afk, but it is line 94

Reply:
Done!